.github/workflows/ci-pip-workflow.yml

```diff
@@ -50,6 +50,7 @@ jobs:
             test-data: package
             matplotlib-pkg: 'matplotlib>=3,<4'
             bokeh-pkg: 'bokeh>=3,<4'
+            arviz-pkg: 'arviz'
```

Comment:
We should really re-evaluate all these builds - and the names they use - but that's for a different PR!